main.js

```diff
@@ -180,6 +180,13 @@
             iconSize: [32, 32],
             iconAnchor: [16, 32],
             popupAnchor: [0, -16]
+        }),
+        NON_TRAFFIC_VIOLATION: L.divIcon({
+            className: 'map-pin darkorchid',
+            html: '<i class="fa fa-road"></i>',
```

Comment:
These are actually _non_ traffic violations, so I don't think a road is fitting. It's a hard one to find an icon for, for sure. What about something generic like this: http://fontawesome.io/icon/sticky-note-o/? So it kind of seems like a citation. We will probably change them all eventually, anyway